imports/plugins/custom/landingPage/client/styles/style.css

```diff
@@ -12,11 +12,12 @@
 
 .header .container {
   margin-top: 22px;
-  min-height: 350px; }
+  min-height: 350px !important; 
```

Comment:
I am guessing this is where the problem is coming from styling a bootstrap class container with important is going to affect all places in the application that makes use of this class

Reply:
hi @Daramola98 , thank you for the observation, on it at the moment.

Reply:
hi @Daramola98 , I found the styling causing the issue, I have fixed it. 
![image](https://user-images.githubusercontent.com/19975842/43309891-fbb09b32-917d-11e8-9caa-3166f28b2ce4.png)